### PR TITLE
"Getting Started" End User Guide

### PR DESCRIPTION
**Pull request by james-mm**
Quick "Getting Started" End User guide to get them up and running on Mattermost quickly.

### CODE REVIEW

File: source/help/getting-started/welcome-to-mattermost.rst [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+Private Channels are for sensitive topics and are only visible to select team members. Any member of a Private Channel can add additional members. Channel members can choose to leave at any time, but only the channel owner or Team Admin can remove other members.
+
+Private Channels have a **lock icon**.
+
+- **Direct Messages (DM)**
+
+Direct Messages are for conversations between two people. 
+
+Group Messages are Direct Messages that have conversations among three or more people. Both are visible only to the people involved.
```
Comment (lindy65) [inline source/help/getting-started/welcome-to-mattermost.rst]: Should we change the heading for this section to "Direct and Group Messages (DM / GM)" ?

**Comment by james-mm** [comment]
Any updates on the status of this request?

**Comment by lindy65** [comment]
Hey @james-mm,

I'm not sure when @esethna will be back so I'm wondering whether @amyblais can help review in the meantime?

**Comment by amyblais** [comment]
@james-mm Good work! I made some edits and checked the doc for grammar and accuracy.

Can you help make some changes to the screenshots? They should have a simple grey border so that they don't bleed into the background and they should not be blurry (a few of them were) - let me know if you would like me to help you with the screenshots or if you have any questions about anything.

**Comment by james-mm** [comment]
@amyblais Thanks! Requested changes made.

**Comment by amyblais** [comment]
@james-mm Thank you! Approved.
@lindy James and I made some edits after your initial editor review, did you want to take another look or is this good to go? Otherwise this is good to merge.

**Pull request merged**